src/components/Modals/DeleteRelationship/DeleteRelModal.jsx

```diff
@@ -0,0 +1,83 @@
+import { useSetAtom, useAtomValue } from "jotai";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import fetchData from "../../../utils/axios";
+import {
+  currentDBIdAtom,
+  userAtom,
+  showDeleteRelationshipModalAtom,
+  deleteTargetRelationshipAtom,
+} from "../../../atoms/atoms";
+
+import Modal from "../../shared/Modal";
+import Button from "../../shared/Button";
+import Content from "../SharedItems/Content";
+import ContentWrapper from "../SharedItems/ContentWrapper";
+import Message from "../SharedItems/Message";
+import Loading from "../../shared/Loading";
+
+function DeleteRelModal() {
```

Comment:
굳이 `DeleteRelModal`이라고 줄여쓰지 않아도 될것 같습니다.
```suggestion
function DeleteRelationshipModal() {
```